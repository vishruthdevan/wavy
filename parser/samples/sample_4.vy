let track1 = [1, 3, 5];
let track2 = [2, 1, -1];

let combined = [track1[0] + track2[0], track1[1] + track2[1], track1[2] + track2[2]];

puts(combined);
