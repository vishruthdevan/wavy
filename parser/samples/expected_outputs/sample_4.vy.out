lexer has 1 error(s):
unterminated string at line 11, position 22
